src/components/Notification/index.js

```diff
@@ -0,0 +1,259 @@
+    switch (variant) {
+      case VARIANT_ERROR:
+        return BG_COLOR_ERROR;
+      case VARIANT_SUCCESS:
+        return BG_COLOR_SUCCESS;
+      default:
+        return BG_COLOR_NOTICE;
+    }
+  }};
+
+  align-items: center;
+  display: flex;
+  margin-left: 50vw;
+  max-width: 1400px;
+  min-height: ${SITE_HEADER_BOTTOM_GAP_HEIGHT}px;
+  position: ${({ top }) =>
+    top === SITE_HEADER_HEIGHT_TALL ? 'absolute' : 'fixed'};
+  top: ${({ top }) => top}px;
+  transform: translateX(-50vw) translateY(0);
+  transition: transform ${SLIDEUP_TIMEOUT}ms ease-out;
+  width: 100vw;
+  will-change: transform;
+  z-index: 1;
+
+  @media (min-width: 1400px) {
+    margin-left: 50%;
+    transform: translateX(-50%) translateY(0);
+  }
+
+  &.slideup {
+    ${({ top }) => css`
+      transform: translateX(-50vw) translateY(-${top}px);
+
+      @media (min-width: 1400px) {
+        transform: translateX(-50%) translateY(-${top}px);
+      }
+    `}
+  }
+
+  @media (min-width: 1400px) {
+    width: 100%;
+  }
+
+  & > * {
+    width: 100%;
+  }
+`;
+
+const Title = styled(Heading).attrs({
+  $as: 'h6',
+})`
+  color: white;
+  font-family: Avenir Next LT W01 Demi, arial, sans-serif;
+  font-weight: normal;
+  margin: ${({ hasMargin }) =>
+    hasMargin
+      ? css`
+          ${themeSpacing(2)} 0 0
+        `
+      : 0};
+
+  & + & {
+    margin-top: ${themeSpacing(2)};
+  }
+`;
+
+const Message = styled(Paragraph)`
+  color: white;
+  margin: 0 0 ${themeSpacing(2)};
+  font-size: 16px;
+`;
+
+const CloseButton = styled(Button)`
+  ${({ alignTop }) =>
+    alignTop
+      ? css`
+          margin-top: ${themeSpacing(2)};
+        `
+      : css`
+          align-self: center;
+        `}
+
+  margin-left: ${themeSpacing(3)};
+
+  &,
+  &:hover {
+    background-color: transparent;
+  }
+
+  & svg path {
+    fill: white !important;
+  }
+`;
+
+/**
+ * Component that shows a title, a close button and, optionally, a message in a full-width bar with
+ * a coloured background. The component slides up automatically after eight seconds, but only when
+ * its variant is not VARIANT_ERROR and its type is not TYPE_GLOBAL.
+ */
+const Notification = ({
+  title,
+  message,
+  onClose,
+  className,
+  type,
+  variant,
+}) => {
+  const [hasFocus, setHasFocus] = useState(false);
+  const [slideUp, setSlideUp] = useState(false);
+  const isFrontOffice = useIsFrontOffice();
+  const tall = isFrontOffice && !isAuthenticated();
+
+  // persisting timeout IDs across renders
+  const onCloseTimeoutRef = useRef();
+  const slideUpTimeoutRef = useRef();
+
+  useEffect(() => {
+    if (variant === VARIANT_ERROR || type === TYPE_GLOBAL) return undefined;
+
+    if (hasFocus) {
+      global.clearTimeout(onCloseTimeoutRef.current);
+      global.clearTimeout(slideUpTimeoutRef.current);
+    } else {
+      const slideUpTimeoutId = global.setTimeout(() => {
+        global.clearTimeout(slideUpTimeoutRef.current);
+
+        setSlideUp(true);
+      }, ONCLOSE_TIMEOUT);
```

Comment:
I have the feeling these 2 names are mixed...

Reply:
Yeah, you're right. I've swapped the names.